Gems/Atom/Feature/Common/Assets/ShaderResourceGroups/Stars/SceneSrg.azsli

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#ifndef AZ_COLLECTING_PARTIAL_SRGS
+#error Do not include this file directly. Include the main .srgi file instead.
+#endif
+
+partial ShaderResourceGroup SceneSrg
+{
+    float3x3 m_starRotationMatrix;
```

Comment:
super nit: naming change? the name indicates connection to star rotation, but if I understand correctly it is really the rotation of the planet you are standing on and from which you inspect the stars.